projects/admin-core/src/lib/shared/components/authorization-edit/authorization-edit.component.ts

```diff
@@ -0,0 +1,145 @@
+import { Component, OnInit, OnDestroy, ChangeDetectorRef } from '@angular/core';
+import { ControlValueAccessor, NG_VALUE_ACCESSOR } from '@angular/forms';
+import { AuthorizationRuleDecision, AuthorizationRuleGroup, GroupModel } from '@tailormap-admin/admin-api';
+import { GroupdetailsService } from '../../../useradmin/services/groupdetails.service';
+import { Subject, takeUntil } from 'rxjs';
+
+@Component({
+  selector: 'tm-admin-authorization-edit',
+  templateUrl: './authorization-edit.component.html',
+  styleUrls: ['./authorization-edit.component.css'],
+  providers: [{
+    provide: NG_VALUE_ACCESSOR,
+    multi: true,
+    useExisting: AuthorizationEditComponent,
+  }],
+})
+export class AuthorizationEditComponent implements OnInit, OnDestroy, ControlValueAccessor {
+  private destroyed = new Subject();
+
+  private _onChange: (_: any) => void = () => undefined;
+  public value: AuthorizationRuleGroup[] = [];
+
+  public newRuleGroup = '';
+  public newRuleDecision = true;
+
+  private groups: GroupModel[] = [];
+
+  public groupList: { name: string; used: boolean }[] = [];
+
+  public selectedChip = 'anonymous';
+  public canAddRule = true;
+
+  constructor(
+    private groupdetailsService: GroupdetailsService,
+    private changeDetectorRef: ChangeDetectorRef,
+  ) {
+  }
+
+  public ngOnInit(): void {
+      this.groupdetailsService.groupList$.pipe(takeUntil(this.destroyed)).subscribe(items => {
+          this.groups = items;
+          this.updateValue(this.value, false);
+      });
+  }
+
+  public ngOnDestroy() {
+      this.destroyed.next(null);
+      this.destroyed.complete();
+  }
+
+
+  private updateValue(value: AuthorizationRuleGroup[], notify: boolean) {
+      this.value = value;
+      this.groupList = this.groups
+        .filter(a => a.name !== 'anonymous' && a.name !== 'app-authenticated')
+        .map(a => ({ name: a.name, used: this.value.find(b => b.groupName === a.name) !== undefined }));
+      this.canAddRule = this.groupList.find(a => !a.used) !== undefined;
+
+      if (notify) {
+          this._onChange(value);
+      }
+
+      // Needed to make sure the initial load of groups works reliably.
+      this.changeDetectorRef.detectChanges();
```

Comment:
I think this should be moved to the end of this function. Now Angular could miss changes to `this.selectedChip`